pram

```diff
@@ -239,6 +249,10 @@ main() {
 
 	local to_close
 	case ${pr} in
+		*://github.com/*/*/pull/*/commits/*)
+			# Simplify for easier use via the api
+			pr=${pr/pull\/*\/commits/commit}
```

Comment:
Sorry for being picky, but I think this implicitly removes appending `.patch` if it's missing.

Reply:
Correct. But `*://github.com/*/*/commit/*` not being handled was true so I handled that in the same branch as `*://github.com/*/*/compare/*`.